src/core/include/openvino/opsets/opset.hpp

```diff
@@ -104,6 +110,13 @@ class OPENVINO_API OpSet {
         return m_op_types;
     }
 
+    template <typename OP_TYPE>
+    std::function<ov::Node*()> get_default_builder() {
```

Comment:
Why do we need this new API?

Reply:
Removed, replaced with inplace lambdas

Reply:
Analysis shows that use of templated function which generates lambdas helps to limit bin size increase because it avoids  duplications

Reply:
Will be moved to private